Sources/SweetTableController.swift

```diff
@@ -24,6 +26,16 @@
             self.addConstraints()
         }
 
+        open override func viewWillAppear(_ animated: Bool) {
+            super.viewWillAppear(animated)
+
+            if self.clearsSelectionOnViewWillAppear {
```

Comment:
>  is available for UITableViewController

😅 right!

Does the end-result of your implementation looks the same as UIKits in the end? 😄

I'd prefer if it defaults to true. I've yet to see the case for preserving selection when popping back.